styles.go

```diff
@@ -67,7 +67,7 @@ type Style struct {
 	// For example "mapbox://fonts/mapbox/{fontstack}/{range}.pbf"
 	Glyphs *string `json:"glyphs,omitempty"`
 	// Layers will be drawn in the order of this array.
-	Layers []json.RawMessage `json:"layers,omitempty"`
+	Layers []*Layer `json:"layers,omitempty"`
```

Comment:
We normally use:
```suggestion
	Layers []Layer `json:"layers,omitempty"`
```

Reply:
In map-service, array of pointers is used for a similar structure. That's why I did it like that.

Reply:
Hm.. yeah. Let's keep it as is for now.